### ISSUE TITLE
Set transparent background for NativeMarkText

### ISSUE THREAD

**Issue by akoshkidko** [andyfinnell/NativeMarkKit#17]
Hi @andyfinnell,

Trying to create `NativeMarkText` with transparent background setting `.backgroundColor(.clear)` in `StyleSheet`. But `NativeMarkText` always has black background, not color of the parent view (i.e. blue). That occurs only with `.clear` background color. When I try it to set `.red` (or any other color) it works fine. 

Am I doing it in a wrong way? 

<img src="https://user-images.githubusercontent.com/16822425/164071224-88a7bbbd-6047-41e9-855b-f8a83b8755a2.jpg" width="300">

**Code example**

```swift
struct ScreenView: View {
    
    var body: some View {
        ScrollView {
            NativeMarkText("Test text Test text Test text Test text Test text Test text Test text Test text Test text Test text Test text Test text Test text Test text Test text", styleSheet: createStyleSheet(appFont: .standard))
        }
        .background(Color.blue)
    }

    
    private func createStyleSheet(appFont: AppFont) -> StyleSheet {
        return StyleSheet.default.mutate(
            block: [
                .document: [
                    .textStyle(.custom(name: .custom(appFont.fontName), size: .fixed(appFont.fontSize))),
                    .textColor(.darkSlateBlue),

                        .backgroundColor(.clear),

                    .orderedListMarker(.arabicNumeral, prefix: "", suffix: "."),
                    .unorderedListMarker(.bullet),
                ],
                .paragraph: [
                    .orderedListMarker(.arabicNumeral, prefix: "", suffix: "."),
                    .unorderedListMarker(.bullet),
                ],
                .thematicBreak: [
                    .thematicBreak(thickness: 1.0, color: UIColor.darkSlateBlue),

                    .orderedListMarker(.arabicNumeral, prefix: "", suffix: "."),
                    .unorderedListMarker(.bullet),
                ],
                .blockQuote: [
                    .textColor(.darkSlateBlue),

                    .orderedListMarker(.arabicNumeral, prefix: "", suffix: "."),
                    .unorderedListMarker(.bullet),
                ],
                .codeBlock: [
                    .orderedListMarker(.arabicNumeral, prefix: "", suffix: "."),
                    .unorderedListMarker(.bullet),
                ],
                .list(isTight: true): [
                    .orderedListMarker(.arabicNumeral, prefix: "", suffix: "."),
                    .unorderedListMarker(.bullet),
                ],
                .list(isTight: false): [
                    .orderedListMarker(.arabicNumeral, prefix: "", suffix: "."),
                    .unorderedListMarker(.bullet),
                ],
                .item: [
                    .orderedListMarker(.arabicNumeral, prefix: "", suffix: "."),
                    .unorderedListMarker(.bullet),
                ]
            ],
            inline: [
                .link: [
                    .textColor(.darkSlateBlue)
                ],
                .code: [
                ],
                .emphasis: [
                ],
                .strong: [
                ],
                .image: [
                ]
            ]
        )
    }
}
```

**Comment by andyfinnell** [comment]
Hi @akoshkidko ,

Thanks the for the report. I can reproduce it and I'm working on a fix. In UIKit `UIColor.clear` is a special case for the background of text, so I'll need to write some extra code to support it.

Kind regards.

**Comment by andyfinnell** [comment]
Fix up in #18

**Comment by andyfinnell** [comment]
Fix has been merged and tagged in the `2.1.1` release.

**Comment by akoshkidko** [comment]
Hi @andyfinnell ,

Thank you for so quick response! 
It works now if I set `background = nil` in `StyleSheet`
